### DISCUSSION TITLE
How to have manually created shapefiles with existing buildings from OpenStreetMap

### DISCUSSION THREAD

**Discussion by Mtyrn** [architecture-building-systems/CityEnergyAnalyst#3252] [Q&A] [answered]
Hi,

How can I open, on the same scenario, existing buildings (by drawing on OpenStreetMap directly on CEA) with manually created buildings (created on QGIS and imported)?

If I draw on OpenStreetMap on CEA first, then I no longer have the option to import manually created buildings.
If I import the created “zone.shp” file from QGIS first, then I no longer have the possibility to draw a section to select existing buildings, and I only have the buildings I created.

Thanks in advance for your help!

Best regards,
Mats

**Comment by lguilhermers** [comment, marked as answer]
Hi Mats,

I suggest you first create the zone by importing it from Open Street Maps directly, then you open the zone.shp file created by it using QGIS. From that, you can edit the shapefile, create new polygons and provide the required attributes so it gets recognized as a building by CEA (make sure you provide new building IDs and include all the attibutes required by CEA). When you are done save the edits so the shapefile changes also in CEA. I also suggesting making a backup before editing it, just in case you need to refer back to the exiting buildings configuration.
If you need help with the second part, this blog post might help you:
https://www.cityenergyanalyst.com/blog-posts/2021/01/20/2021-1-20-how-to-add-own-building-geometries-to-a-scenario-import-of-manually-created-shapefiles

**Reply by Mtyrn** [reply]
Hi,

Thank you for your answer. Works perfectly! :)
